lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -110,6 +122,22 @@ export default class BitcoinProcessor {
         config.bitcoinWalletImportString,
         config.requestTimeoutInMilliseconds || 300,
         config.requestMaxRetries || 3);
+
+    this.lockResolver = new LockResolver(this.bitcoinClient);
+
+    this.mongoDbLockTransactionStore = new MongoDbLockTransactionStore(config.mongoDbConnectionString, config.databaseName);
+
+    const valueTimeLockTransactionFeesInBtc = config.valueTimeLockTransactionFeesAmountInBitcoins || 0.25;
+    const numberOfBlocksInOneMonth = BitcoinClient.estimatedNumberOfBlocksInOneHour * 24 * 30;
+
+    this.lockMonitor =
+      new LockMonitor(
+        this.bitcoinClient,
+        this.mongoDbLockTransactionStore,
+        config.valueTimeLockPollPeriodInSeconds || 10 * 60,
+        config.valueTimeLockAmountInBitcoins * BitcoinProcessor.satoshiPerBitcoin, // Desired lock amount in satoshis
+        numberOfBlocksInOneMonth,                                                  // Desired lock duration in blocks
+        valueTimeLockTransactionFeesInBtc * BitcoinProcessor.satoshiPerBitcoin);   // Txn Fees amoount in satoshis
```

Comment:
Any reason why we cannot make internal code and config both use satoshis? Also, would be nice if the lockTransactionsFee is right after lockAmount.

Reply:
I think it's easier for the admin to think in BTC instead of figuring out the BTC and then multiplying w/ the right value to convert to satoshis etc.